cv/cv/src/cv_utils/camera_projection.py

```diff
@@ -3,8 +3,11 @@
 
 import cv2
 import numpy as np
+import rospy
+from image_geometry import PinholeCameraModel
 import rospkg
 
+from sensor_msgs.msg import CameraInfo
 
 class CameraProjection:
     def __init__(self):
```

Comment:
A slight nit, technically we should be subscribing to camera info continuously and updating the camera model continuously but I'm ok with how it is here since the intrinsics don't change during our execution anyways